src/main/java/org/terasology/launcher/gui/javafx/ApplicationController.java

```diff
@@ -501,6 +512,15 @@ private void updateButtons() {
             downloadButton.setVisible(true);
             cancelDownloadButton.setVisible(false);
         }
+
+        // if less than 200MB available
+        if (downloadDirectory.getUsableSpace() <= 200 * 1000000) {
```

Comment:
in any case, we should get rid of the *magic numbers* (I don't mind the small error induced by mega vs mebibyte). Maybe use something like 
```private static final int MINIMAL_FREE_SPACE_IN_MB = 200 * 1000 * 1000 ```